variants/STM32MP157_DK/README.md

```diff
@@ -13,49 +13,63 @@ Because every OS may have different software configurations (especially Device T
         https://github.com/stm32duino/Arduino_Core_STM32/tree/master/variants/STM32MP157_DK/README.md
 
-In this example, the user **must** upload `<Arduino build output path>/run_arduino.sh` file manually. Uploading instruction is described later in the [Uploading](#Uploading) section. 
+`<Arduino build output path>/run_arduino_<your project name>.sh` looks like this for *Blink* example:
 
-After uploading the user can use `sh run_arduino.sh start` in the console of host Linux via either SSH or Serial Console, to run the Arduino firmware.
+* Windows:
 
-Note: `sh run_arduino.sh start` is a one-shot command: the Arduino firmware only runs for the current boot. If you want to make it run after reboot, you need to use `sh run_arduino.sh install` command. 
+  `C:/Users/%USERNAME%/AppData/Local/Temp/arduino_build_668148/run_arduino.Blink.sh`
 
-`run_arduino.sh` help page summary:
+* Linux/macOS:
 
-    Usage: sh run_arduino.sh [start|stop|restart|generate|install|uninstall]
+  `/tmp/arduino_build_668148/run_arduino_Blink.sh`
 
-    run_arduino.sh is a helper script that helps managing an Arduino binary
+In this example, the user **must** upload `<Arduino build output path>/run_arduino_<sketch name>.sh` file manually. Uploading instruction is described later in the [Uploading](#Uploading) section. 
+
+After uploading the user can use `sh run_arduino_<sketch name>.sh start` in the console of host Linux via either SSH or Serial Console, to run the Arduino firmware.
+
+#### Note
+ * `sh run_arduino_<sketch name>.sh start` is a one-shot command, the Arduino firmware only runs for the current boot. If you want to make it run after reboot, you need to use `sh run_arduino_<sketch name>.sh install` command. 
+
+`run_arduino_<sketch name>.sh` help page summary:
+
+    Usage: sh run_arduino_<sketch name>.sh [start|stop|restart|generate|install|uninstall]
+
+    run_arduino_<sketch name>.sh is a helper script that helps managing an Arduino binary
     file for the coprocessor using remoteproc framework.
 
-    sh run_arduino.sh start
+    sh run_arduino_<sketch name>.sh start
         Upload the binary to the coprocessor then start it.
         This command must be executed while the script contains the binary
         after generate command is run.
 
-    sh run_arduino.sh install
+    sh run_arduino_<sketch name>.sh install
         Run the binary on boot automatically by installing a systemd service.
 
-    sh run_arduino.sh uninstall
+    sh run_arduino_<sketch name>.sh uninstall
         Uninstall the autostart service.
 
-    sh run_arduino.sh stop
+    sh run_arduino_<sketch name>.sh stop
         Stop the coprocessor.
 
-    sh run_arduino.sh restart
+    sh run_arduino_<sketch name>.sh restart
         Restart the coprocessor.
 
-See the source code for the full help page and the more details about [run_arduino.sh].
+See the source code for the full help page and the more details about [run_arduino_<sketch name>.sh].
```

Comment:
It looks like `<>` in link does not work, rendering like `run_arduino_`.
```suggestion
See the source code for the full help page and the more details about [run_arduino_gen.sh].
```

Reply:
I've updated and add a sentence about the gen script to be more precise.